src/main/java/org/opentripplanner/apis/gtfs/datafetchers/QueryTypeImpl.java

```diff
@@ -325,7 +326,7 @@ public DataFetcher<Connection<PlaceAtDistance>> nearest() {
         : null;
       List<PlaceType> filterByPlaceTypes = args.getGraphQLFilterByPlaceTypes() != null
         ? args.getGraphQLFilterByPlaceTypes().stream().map(GraphQLUtils::toModel).toList()
-        : null;
+        : EnumSet.complementOf(EnumSet.of(PlaceType.STATION)).stream().toList();
```

Comment:
Please make this a constant of `PlaceType`.

Reply:
And use `List.copyOf` to make it immutable.

Reply:
@leonardehrenfried The returned value is a immutable list, is that not ok?

Reply:
It's created anew for every request when it could be a constant.

Reply:
I didn't say it very clearly I wanted to get rid of intermediate stream creation, which is not needed.